man/systemd.netdev.xml

```diff
@@ -342,6 +342,24 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>GroupForwardMask=</varname></term>
+          <listitem>
+            <para>Bitmask to allow forwarding of link local frames with address
+            01-80-C2-00-00-0X on a bridge device. Only values that set bits
+            not matching BR_GROUPFWD_RESTRICTED in net/bridge/br_private.h allowed.
```

Comment:
Uh, documentation should not contain references to C code files. This is admin/user facing stuff, please document this in a way people who are not C hackers have a chance to grok this. It doesn't have to be perfect, but please, this is a bit too sloppy...

Reply:
How does this read:

This setting controls the IFLA_BR_GROUP_FWD_MASK option, a 16-bit bitmask represented as an integer which allows forwarding of link local frames with select 802.1D reserved addresses (01:80:C2:00:00:0X). A logical AND is performed between the specified bitmask and the exponentiation of 2^X, the lower nibble of the last octet of the destination. For example, a value of 8 would allow forwarding of frames addressed to 01:80:C2:00:00:03 commonly used by 802.1X PAE.